### DISCUSSION TITLE
Keep selected state after duplicateShapes

### DISCUSSION THREAD

**Discussion by txh0001** [wkentaro/labelme#1399] [Ideas / Feature requests]
when i use duplicateShapes feature ,all shapes will lost select state after i copy them. so i can not move them to a new position. it seems that when i comment the follow code ,it is worked as i want:

    def duplicateSelectedShape(self):
        added_shapes = self.canvas.duplicateSelectedShapes()
        # self.labelList.clearSelection()   #comment this
        for shape in added_shapes:
      ..........
so could keep this change on next version?

**Comment by wkentaro** [comment]
Makes sense. Merged to main via this PR: https://github.com/labelmeai/labelme/pull/1401
